src/Splat.Autofac/AutofacDependencyResolver.cs

```diff
@@ -94,14 +94,14 @@ public virtual IEnumerable<object> GetServices(Type serviceType, string? contrac
 
                     if (instance is null)
```

Comment:
this is always false as Resolve returns non-nullable

Reply:
wip in https://github.com/reactiveui/splat/commit/f50a07287b0350543257ccd7370e5ded5b38e196